### PR TITLE
Updating README/setup.py for new location.

### PR DESCRIPTION
**Pull request by dhermes**
- Based on https://github.com/GoogleCloudPlatform/google-auth-library-python/issues/167
- Fixed "and and" typo
- Fixed URL for project
- Updated the package description to explain it was for `httplib2` transport
- Added `httplib2` as a dependency

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 DEPENDENCIES = (
     'google-auth'
```
Comment (jonparrott) [inline setup.py]: You need a comma here, travis is barfing. :)

**Comment by jonparrott** [comment]
LGTM and thanks for doing this, @dhermes. You just gotta add that final comma and Travis should be happy.

File: setup.py (hunk 1 above)
Comment (dhermes) [inline setup.py]: Haha. DERP city.

**Comment by dhermes** [comment]
Just `--amend`-ed and did a force push. (I don't have merge privileges, or any sort of privileges in this repo.)

**Comment by jonparrott** [comment]
@dhermes you do now. :)

**Pull request merged**